members/admin/user_admin.py

```diff
@@ -139,3 +142,24 @@ def get_list_filter(self, request):
                 AdminUserUnionListFilter,
                 AdminUserDepartmentListFilter,
             ]
+
+    def get_fieldsets(self, request, obj=None):
+        fieldsets = super(UserAdmin, self).get_fieldsets(request, obj)
+        if not obj:
+            return fieldsets
+        if not request.user.is_superuser:
+            fieldsets = deepcopy(fieldsets)
```

Comment:
Kan du bruge et andet navn end `fieldsets` her? det forvirrede mig noget at `fieldsets` variablen sættes flere gange, og tilsyneladende bliver brugt til forskellige ting? hvad indeholder det fieldset rent faktisk? rettigheder for brugeren?

Reply:
Talte med Mikael offline om dette. `fieldsets` indeholder de fields der bliver vist på siden, så giver mening. ser ok ud med mig